index.html

```diff
@@ -0,0 +1,296 @@
+<section class="about-me">
+        <div class="about-myself">
+          
+          <div class="about-content">
+          <h1 >About Me</h1>
+          <div class="about-imgs">
+            <img class="img1" src="./images/frame4.png" alt="right side image">
+            <img class="img2" src="./images/frame5.png" alt="left side image">
+          </div>
+          <p class="about-support-text">
+             I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+          </p>
+           <button class="project">Get my Resume</button>
+            </div>
+          </div >
+         <section class= "grids">
+         <div class="grid">
+            <img class="grid-image" src="./images/language.png" alt="">
+            <div class="txt1">
+            <h3 class="lang-title">Languages</h3>
+             <section class = "tags1">
+            <li class="tag1">Javascript</li>
+            <li class="tag1">Ruby</li>
+            <li class="tag1">Html</li>
+            <li class="tag1">CSS</li>
+          </section>
+          </div>
+         </div>
+          <div class="grid">
+          <img  class="grid-image" src="./images/frameworks.png" alt="">
+          <div class="txt1">
+          <h3>Frameworks</h3>
+           <div class = "tags1">
+            <li class="tag1">Bootstrap</li>
+            <li class="tag1">Ruby on Rails</li>
+            <li class="tag1">Rspec</li>
+           </div>
+             <div class = "tags2">
+            <li class="tag1">Capybara</li>
+            <li class="tag1">Selenium</li>
+          </div>
+        </div>
+          </div>
+       <div class="grid">
+        <img class="grid-image"src="./images/skills.png" alt="">
+        <div class="txt1">
+        <h3>Skills</h3>
+        <div class = "tags1">
+           <li class="tag1">Codekit</li>
+            <li class="tag1">Github</li>
+            <li class="tag1">Codepen</li>
+            <li class="tag1">Gitlab</li>
+  </div>
+            <div class="tags2">
+            <li class="tag1">Terminal</li>
+            </div>
+          </div>
+     </div> 
+     
+         </section>
+         <div class="contact-form">
+    <div class="form-text">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+   <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xpznodvr" method="post">
+  <ul>
+    <li>
+    <input type="text" name="fname" placeholder="Full name" maxlength="30" required>
+  </li>
+  <li>
+    <input type="email" name="email" placeholder="Email address" required>
+  </li>
+  <li>
+    <textarea name="message" placeholder="Enter your text here" maxlength="500" required></textarea>
+  </li>
+  <button type="submit" name="button" >Get in touch</button>
+  </ul>
+</form>
+<div class="form-images">
```

Comment:
- [ ] Just as the previous reviewer pointed out, there are still some discrepancies in the form section. You will notice that the content beside the form is not aligned according to Figma. Check the Figma the previous reviewer posted.
![image](https://user-images.githubusercontent.com/86200356/200264640-a79b5967-bc2d-455a-b897-824372865473.png)